docs/admin.md

```diff
@@ -1,5 +1,5 @@
 # Admin Dashboard
 
 The Flarum Admin Dashboard is a user-friendly interface for managing your forum.
-It is only available to users in the "Admin" [group](permissions.md).
+It is only available to users in the "Admin" [group](/extend/permissions.md).
```

Comment:
The original is actually correct, we just haven't managed to get around to writing the documentation for that yet.